crypto/x509/x509_test.cc

```diff
@@ -3601,99 +3507,6 @@ TEST(X509Test, CommonNameFallback) {
                         "foo.host1.test"));
 }
 
-TEST(X509Test, LooksLikeDNSName) {
-    static const char *kValid[] = {
-        "example.com",
-        "eXample123-.com",
-        "*.example.com",
-        "exa_mple.com",
-        "example.com.",
-        "project-dev:us-central1:main",
-    };
-    static const char *kInvalid[] = {
-        "-eXample123-.com",
-        "",
-        ".",
-        "*",
-        "*.",
-        "example..com",
-        ".example.com",
-        "example.com..",
-        "*foo.example.com",
-        "foo.*.example.com",
-        "foo,bar",
-    };
```

Comment:
Could some of the test cases be repurposed to test either `cn2dnsid` or `equal_wildcard` (both of which are currently `static` so not directly tested)?